### PR TITLE
added from uint8list

### PR DESCRIPTION
**Pull request by serverplumber**
I needed qrcodes which display binary data so I wrote this little chunk which bypasses the conversions in `QrByte`

### CODE REVIEW

**Comment by kevmoo** [comment]
Need to format the code. https://travis-ci.org/github/kevmoo/qr.dart/jobs/698701045

(Not sure why travis PR notices are missing here)

Also please add an entry to the changelog!

**Comment by serverplumber** [comment]
For the code formatting : Will a plain `dartfmt -w` do it?

I'm not sure what to put in the changelog will this do?
 -  Support for creating a `QrCode` from a `Uint8List` for binary qr codes

I was about to create a pr for [QrErrorCorrectionLevel](https://github.com/kevmoo/qr.dart/issues/30) so you might want to wait a bit for the version bump.

File: test/qr_code_test.dart [hunk 1]
```diff
@@ -29,6 +31,19 @@ void main() {
...
+
+  test('fromUint8List', () {
+    for (var quality in QrErrorCorrectLevel.levels) {
+      final qr = QrCode.fromUint8List(
+          data: Uint8List.fromList([115, 104, 97, 110, 110, 97, 33]),
+          errorCorrectLevel: quality)
+        ..make();
+      for (var i = 0; i < qrModules(qr).length; i++) {
+        expect(_encodeBoolListToString(qrModules(qr)[i]),
+            qrCodeTestData[1.toString()][quality.toString()][i]);
```
Comment (kevmoo) [inline test/qr_code_test.dart:43]: `1.toString` is a bit weird here – how about just `'1'`?

Comment (serverplumber) [reply test/qr_code_test.dart:43]: I thought the same thing when I read line 30 of your code.

**Comment by kevmoo** [comment]
Ha! This is pretty old!

**Comment by serverplumber** [comment]
Would you like me to change both?

**Comment by kevmoo** [comment]
I fix it!

**Comment by serverplumber** [comment]
Awesome thanks!

**Pull request merged**